src/migrations/versions/906d7198df28_privacy_request_approve.py

```diff
@@ -17,8 +17,9 @@
 
 
 def upgrade():
-    op.execute("alter type privacyrequeststatus add value 'approved'")
-    op.execute("alter type privacyrequeststatus add value 'denied'")
+    with op.get_context().autocommit_block():
```

Comment:
This is needed in order to prevent the error in: https://github.com/ethyca/fidesops/runs/5778340779?check_suite_focus=true

New Enum values must be committed to the DB before they can be used. Since Alembic is efficiently committing transactions, they aren't automatically committed without the line so subsequently can't be added by the next migration in the tree.